playbooks/acs.yml

```diff
@@ -742,3 +759,37 @@
         mode: "0755"
   tags:
     - audit_storage
+
+- name: HxInsight Role
+  hosts: hxi
+  gather_facts: false
+  pre_tasks:
+    - name: Assert HxI is available
+      ansible.builtin.fail:
+        msg: "HxInsight can be used only with ACS Enterprise"
+      when: not acs_is_enterprise
+    - name: Include HxInsight vars
+      ansible.builtin.include_vars:
+        file: ../vars/hxi.yml
```

Comment:
It's probably safe to use a `var_files` at the play level instead or plain `set_facts` tasks

Reply:
here could be possible as the conditional in the task is not necessary, but maybe I prefer staying with the task to be consistent with the repository play